LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Copyright 2004-2016 Castle Project - http://www.castleproject.org/
```

Comment:
For the LICENSE file issue, the fix is to use this in common.props:
`<Content Include="..\..\LICENSE"> <PackagePath></PackagePath> </Content>
`
https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/issues/197